robot.py

```diff
@@ -89,8 +95,9 @@ async def raise_hand(self):
         that wiggles the hand on inactivity.
         """
         if self._wiggler is not None:
-            print("LOGIC BUG: trying to raise already-raised hand")
+            self.logger.warning("LOGIC BUG: trying to raise already-raised hand")
             return
+        self.logger.info("raise hand")
```

Comment:
The default log level is info, and this will be printed out every time. I'm inclined to either make the default level warning or make most logging debug, but I'm open to disagreement.

Reply:
Good point. I did this on purpose in case we DID want this to be printed, but if you don't believe so, then debug it is!